### PR TITLE
Del 300 product downtime

### PR DESCRIPTION
**Pull request by wei-lee**
https://issues.redhat.com/browse/DEL-300

to verify:

1. Prepare  a RHMI cluster and have the workload app installed.
2. Run the following command from this branch:
   ```
   ./delorean pipeline query-report \
      --config-file ./configurations/downtime-report-config.yaml \
      -o /tmp
   ```
3. Once finished, check the output file

### CODE REVIEW

**Comment by b1zzu** [comment]
:eyes:

File: cmd/productTests.go [hunk 1]
```diff
@@ -58,8 +58,8 @@ type runTestsCmd struct {
 func init() {
 	f := &runTestsCmdFlags{}
 	cmd := &cobra.Command{
-		Use:   "run",
-		Short: "Execute test containers",
+		Use:   "product-tests",
```
Comment (b1zzu) [inline cmd/productTests.go:61]: We should look where the `delorean tests run` is documented and update it

Comment (b1zzu) [reply cmd/productTests.go:61]: we should fix the Makefile in the integreatly-operator otherwise the nightly pipelines will fails

**Comment by b1zzu** [comment]
/lgtm

But we should fix or at least have a PR for the integreatly-operator before merging this one

File: cmd/productTests.go (hunk 1 above)
Comment (wei-lee) [reply cmd/productTests.go:61]: yes, I will do that

**Comment by wei-lee** [comment]
@b1zzu need to rebase so if you can approve again that will be great. Also I linked it to operator PR to update the make target too. Thanks.

**Comment by STEPHINRACHEL** [comment]
Verified locally 
The output file generated can be found [here](https://gist.github.com/STEPHINRACHEL/b72dac26143939f3c0e424a627427fa3)
/approve

**Pull request merged**